### PR TITLE
fix corerun hardcoding to netcoreapp1.1

### PR DESCRIPTION
**Pull request by ianhays**
@ericstj @weshaggard @karajas

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets [hunk 1]
```diff
@@ -21,17 +21,17 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <TestWithCore Condition="'$(TestWithCore)' == '' And '$(TestTFM)' != 'netcoreapp1.0'">false</TestWithCore>
+    <TestWithCore Condition="'$(TestWithCore)' == '' And !$(TestTFM.Contains('netcoreapp'))">false</TestWithCore>
```
Comment (ericstj) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets]: StartsWith('netcoreapp', StringComparison.OrdinalIgnoreCase)

**Comment by ericstj** [comment]
LGTM modulo minor comment.

File: src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets (hunk 1 above)
Comment (ianhays) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets]: yeah, that'd be better. updated.

**Comment by ianhays** [comment]
Thanks for taking a look Eric.

**Pull request merged**